configs/config_all.yaml

```diff
@@ -111,6 +111,10 @@ process:
       tokenization: false                                     # Whether to count the ratio of alphanumeric to the total number of tokens.
       min_ratio: 0.0                                          # the min ratio of filter range
       max_ratio: 0.9                                          # the max ratio of filter range
+  - audio_duration_filter:                                  # Keep data samples whose audios' durations are within a specified range.
+      min_duration: 0                                         # the min audio duration of filter range
+      max_duration: 3600                                      # the max audio duration of filter range
```

Comment:
What is the unit of duration, I guess seconds?  Suggest documenting it more clear (as well as other places).

Reply:
👍🏻Yes, it's in seconds. The unit information has been added in the latest commit.